components/properties/src/sets.rs

```diff
@@ -76,10 +117,24 @@ impl CodePointSetData {
         }
     }
 
-    // TODO: we should instead integrate better with UnicodeSetBuilder
-    #[doc(hidden)]
-    pub fn as_unicodeset(&self) -> &'_ UnicodeSet<'_> {
-        &self.data.get().inv_list
+    /// Construct a new one an owned [`UnicodeSet`]
+    pub fn from_unicode_set(set: UnicodeSet<'static>) -> Self {
```

Comment:
Nit:

- `from_static_unicode_set`
- `from_owned_unicode_set`
- `from_unicode_set_owned`

Reply:
I don't think any other API is possible due to the yoke?

Reply:
This comment contains suggestions for the name of this function.

Reply:
Right, I'm saying that it's unclear to me if we need a more specific name since this is the only way a from API could work